package/cloudshell/orch/training/logic/create_user_sandboxes.py

```diff
@@ -55,11 +55,11 @@ def _get_latest_sandbox_details(self, sandbox: Sandbox) -> ReservationDescriptio
         return sandbox_details.ReservationDescription
 
     def _send_emails(self):
-        if self._email_service.is_email_configured():
-            for user in self._env_data.users_list:
-                student_link = self._users_data.get_key(user, userDataKeys.STUDENT_LINK)
-                self._email_service.send_email(user, student_link)
-                self._sandbox_output.notify(f'Sending email to {user} with link={student_link}')
+        # if self._email_service.is_email_configured():
+        for user in self._env_data.users_list:
+            student_link = self._users_data.get_key(user, userDataKeys.STUDENT_LINK)
+            self._email_service.send_email([user], 'Welcome to Training', template_parameters={'sandbox_link': student_link})
```

Comment:
if I want to use a custom template and subject for the training email, how will that work?

Reply:
the send_email function takes an optional parameter 'template_name' which will load that file instead of using the default.

Reply:
I understand that but at the moment it is not usable for the blueprint designer who uses the training workflow. Because all params that you pass to this method are basically hard coded.
We need to have a way to override the following defaults:
- email subject
- template name 
- template parameters
makes sense?